src/SearchInput/SearchInput.stories.js

```diff
@@ -9,4 +9,18 @@ export default {
 
 const Template = (args) => <Container {...args} />
 
+const SearchInputWithOutline = () => (
+  <SearchInput
+    id="days"
+    label="Days"
+    placeholder="Search days"
+    searchList={list}
+    onFound={() => {}}
+    showCaret
+    outlined
+  />
+)
+
```

Comment:
instead of duplicating this one you should have added the props in `Container.tsx` so if we even need to add others we can just expand on that instead of creating duplicates of the same component